packages/keystone/package.json

```diff
@@ -79,6 +79,7 @@
     "serialize-error": "^8.1.0",
     "supertest": "^6.1.3",
     "tough-cookie": "^4.1.3",
+    "undici": "5.28.4",
```

Comment:
Plus 1 library for send requests, now there are 5 of them + 1 our custom))

Reply:
node-fetch was removed from condo and packages/keystone